apps/sms/test/marionette/lib/views/activity-chooser/view.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+/* global module */
+
+var MenuAccessor = require('../shared/menu_accessors');
```

Comment:
We've discussed ```activity-chooser``` with Johan some time ago - so here is the first implementation of what I have in mind.